bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/RecaptchaImpl.java

```diff
@@ -97,6 +92,7 @@ public String getSize() {
     }
 
     @Override
+    @JsonIgnore
     public String getProvider() {
```

Comment:
Same here, it should be part of interface. Fix it at other places as well

Reply:
done